docs/maintaining.md

```diff
@@ -0,0 +1,22 @@
+# Maintaining Notes
+
+This document captures informal policies, tips, and topics useful to maintaining the Rust track.
+
+## Internal Tooling
+
+We have a number of scripts for CI tests.
+They live in `bin/` and `_test/`.
+
+## Internal Tooling Style Guide
+
+This is non-exhaustive.
+
+- Prefer snakecase for file names
+    ```sh
+    hello_world.sh
+    ```
+- Prefer BASH for scripting
+- Adopt a Unix philosophy for tooling
+  - try to accomplish tooling needs using `jq` and pipelines of minimal utilities
```

Comment:
```suggestion
  - prefer using tools that do one thing well
  - prefer using tools that are ubiquitous: `jq` or `sed` instead of `prettier` or `sd`
  - write scripts to do one thing well
```

Expanding on the philosophy a bit.